src/vaadin-grid-data-provider-mixin.html

```diff
@@ -123,6 +143,10 @@
           reflectToAttribute: true
         },
 
+        /**
+         * @type {ItemCache}
```

Comment:
This seems to always have an `ItemCache` instance.
```suggestion
         * @type {!ItemCache}
```

Reply:
Done.